pkg/internal/computeresources/transformer_test.go

```diff
@@ -1140,7 +1140,7 @@ func TestTransformerMultipleContainer(t *testing.T) {
 					},
 					Requests: corev1.ResourceList{
 						corev1.ResourceCPU:              resource.MustParse("1"),
-						corev1.ResourceMemory:           resource.MustParse("400Mi"),
+						corev1.ResourceMemory:           resource.MustParse("200Mi"),
```

Comment:
I think there was another comment thread about this, but this line should stay "400Mi".